modules/sdk-coin-coredao/src/coredao.ts

```diff
@@ -31,4 +32,46 @@ export class Coredao extends AbstractEthLikeNewCoins {
     const explorerUrl = common.Environments[this.bitgo.getEnv()].coredaoExplorerBaseUrl;
     return await recoveryBlockchainExplorerQuery(query, explorerUrl as string, apiToken);
   }
+
+  /** @inheritDoc */
+  async queryAddressBalance(address: string): Promise<BN> {
+    const result = await this.recoveryBlockchainExplorerQuery({
+      module: 'account',
+      action: 'balance',
+      address: address,
+    });
+    // throw if the result does not exist or the result is not a valid number
+    if (!result || !result.result || isNaN(<number>result.result)) {
+      throw new Error(`Could not obtain address balance for ${address} from the explorer, got: ${result.result}`);
+    }
+
+    if (typeof result.result !== 'string') {
+      result.result = result.result.toString();
+    }
```

Comment:
```suggestion
      result.result = result.result.toString();
```
you could just unconditionally do this.